### PR TITLE
Kamenetsky formula

### PR DESCRIPTION
**Pull request by khusheekapoor**
For issue #649 as a GSSoC'21 Participant.

### CODE REVIEW

File: Code/Java/Kamenetsky_Formula.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
+
+/*
+ * The Kamenetsky Formula is used to calculate the number of digits in the factorial of a number.
+ * Formula:
+ *        number of digits in factorial = log10( ((n/e)^n) * sqrt(2*pi*n))
```
Comment (rish-singhal) [inline Code/Java/Kamenetsky_Formula.java:5]: can you add a proof to the formula? or add some reference link??

File: Algorithm/Number_theory/readme.md [hunk 2]
```diff
@@ -39,3 +39,4 @@
 # Questions related to Number Theory
 
 - GCD of two number using Euclidean Algorithm ----> [Java](/Code/Java/GCD.java)
+- Counting number of digits in factorial using Kamenetsky's Formula ----> [Java](Code/Java/Kamenetsky_Formula.java)
```
Comment (rish-singhal) [inline Algorithm/Number_theory/readme.md:42]: add the name in the sorted alphabetical order, i.e. Counting.... should come before GCD...

**Review by rish-singhal — CHANGES_REQUESTED** [review]

File: Algorithm/Number_theory/readme.md (hunk 2 above)
Comment (khusheekapoor) [reply Algorithm/Number_theory/readme.md:41]: made the change.

File: Code/Java/Kamenetsky_Formula.java (hunk 1 above)
Comment (khusheekapoor) [reply Code/Java/Kamenetsky_Formula.java:5]: made the changes.

**Pull request merged**